test/vg-we.supp

```diff
@@ -0,0 +1,525 @@
+{
+   <insert_a_suppression_name_here>
```

Comment:
You should explicitly name these to let people know we're ignoring `uninitialized value` errors